diff --git a/dlls/appwiz.cpl/addons.c b/dlls/appwiz.cpl/addons.c
index 57855648ff1..52a5b745d0a 100644
--- a/dlls/appwiz.cpl/addons.c
+++ b/dlls/appwiz.cpl/addons.c
@@ -110,7 +110,7 @@ static LPWSTR url = NULL;
 static IBinding *dwl_binding;
 static WCHAR *msi_file;
 
-extern const char * CDECL wine_get_version(void);
+extern const char * CDECL wien_get_version(void);
 
 static WCHAR * (CDECL *p_wine_get_dos_file_name)(const char*);
 
@@ -612,7 +612,7 @@ static void append_url_params( WCHAR *url )
     len += MultiByteToWideChar(CP_ACP, 0, addon->version, -1, url+len, size/sizeof(WCHAR)-len)-1;
     lstrcpyW(url+len, L"&winev=");
     len += lstrlenW(L"&winev=");
-    MultiByteToWideChar(CP_ACP, 0, wine_get_version(), -1, url+len, size/sizeof(WCHAR)-len);
+    MultiByteToWideChar(CP_ACP, 0, wien_get_version(), -1, url+len, size/sizeof(WCHAR)-len);
 }
 
 static LPWSTR get_url(void)
diff --git a/dlls/ntdll/ntdll.spec b/dlls/ntdll/ntdll.spec
index b7659ac8c49..a7ba7139ead 100644
--- a/dlls/ntdll/ntdll.spec
+++ b/dlls/ntdll/ntdll.spec
@@ -1713,7 +1713,7 @@
 @ cdecl -norelay __wine_dbg_strdup(str)
 
 # Version
-@ cdecl wine_get_version()
+@ cdecl wien_get_version()
 @ cdecl wine_get_build_id()
 @ cdecl wine_get_host_version(ptr ptr)
 
diff --git a/dlls/ntdll/version.c b/dlls/ntdll/version.c
index 9c396598d4e..20ffa872b8c 100644
--- a/dlls/ntdll/version.c
+++ b/dlls/ntdll/version.c
@@ -216,9 +216,9 @@ static const RTL_OSVERSIONINFOEXW *current_version;
 static char wine_version[256];
 
 /*********************************************************************
- *                  wine_get_version
+ *                  wien_get_version
  */
-const char * CDECL wine_get_version(void)
+const char * CDECL wien_get_version(void)
 {
     return wine_version;
 }
diff --git a/dlls/user32/nonclient.c b/dlls/user32/nonclient.c
index 00c7368c516..ec4424211fe 100644
--- a/dlls/user32/nonclient.c
+++ b/dlls/user32/nonclient.c
@@ -159,10 +159,10 @@ LRESULT NC_HandleSysCommand( HWND hwnd, WPARAM wParam, LPARAM lParam )
             if (hmodule)
             {
                 BOOL (WINAPI *aboutproc)(HWND, LPCSTR, LPCSTR, HICON);
-                extern const char * CDECL wine_get_version(void);
+                extern const char * CDECL wien_get_version(void);
                 char app[256];
 
-                sprintf( app, "Wine %s", wine_get_version() );
+                sprintf( app, "Wine %s", wien_get_version() );
                 aboutproc = (void *)GetProcAddress( hmodule, "ShellAboutA" );
                 if (aboutproc) aboutproc( hwnd, app, NULL, 0 );
                 FreeLibrary( hmodule );
diff --git a/dlls/wined3d/adapter_vk.c b/dlls/wined3d/adapter_vk.c
index 5b6042659aa..fd5d2df6652 100644
--- a/dlls/wined3d/adapter_vk.c
+++ b/dlls/wined3d/adapter_vk.c
@@ -1902,14 +1902,14 @@ static const struct wined3d_adapter_ops wined3d_adapter_vk_ops =
 
 static unsigned int wined3d_get_wine_vk_version(void)
 {
-    const char * (CDECL *wine_get_version)(void) = (void *)GetProcAddress( GetModuleHandleW(L"ntdll.dll"),
-                                                                           "wine_get_version" );
+    const char * (CDECL *wien_get_version)(void) = (void *)GetProcAddress( GetModuleHandleW(L"ntdll.dll"),
+                                                                           "wien_get_version" );
     const char *ptr;
     int major, minor;
 
-    if (!wine_get_version) return VK_MAKE_VERSION(1, 0, 0);
+    if (!wien_get_version) return VK_MAKE_VERSION(1, 0, 0);
 
-    ptr = wine_get_version();
+    ptr = wien_get_version();
     major = atoi(ptr);
 
     while (isdigit(*ptr))
diff --git a/programs/winecfg/about.c b/programs/winecfg/about.c
index 0b860bb1955..19f63c8120a 100644
--- a/programs/winecfg/about.c
+++ b/programs/winecfg/about.c
@@ -38,7 +38,7 @@ static HFONT titleFont = NULL;
 INT_PTR CALLBACK
 AboutDlgProc (HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
 {
-    const char * (CDECL *wine_get_version)(void);
+    const char * (CDECL *wien_get_version)(void);
     HWND hWnd;
     HDC hDC;
     RECT rcClient, rcRect;
@@ -110,8 +110,8 @@ AboutDlgProc (HWND hDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
                                  0, 0, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, L"Tahoma" );
         SendDlgItemMessageW(hDlg, IDC_ABT_TITLE_TEXT, WM_SETFONT, (WPARAM)titleFont, TRUE);
 
-        wine_get_version = (void *)GetProcAddress( GetModuleHandleW(L"ntdll.dll"), "wine_get_version" );
-        if (wine_get_version) SetDlgItemTextA(hDlg, IDC_ABT_PANEL_TEXT, wine_get_version());
+        wien_get_version = (void *)GetProcAddress( GetModuleHandleW(L"ntdll.dll"), "wien_get_version" );
+        if (wien_get_version) SetDlgItemTextA(hDlg, IDC_ABT_PANEL_TEXT, wien_get_version());
 
         ReleaseDC(hDlg, hDC);
 
diff --git a/programs/winver/winver.c b/programs/winver/winver.c
index ee3120443eb..a1cee5d1598 100644
--- a/programs/winver/winver.c
+++ b/programs/winver/winver.c
@@ -25,11 +25,11 @@
 int PASCAL WinMain (HINSTANCE inst, HINSTANCE prev, LPSTR cmdline, int show)
 {
     char name[128] = "Wine ";
-    const char * (CDECL *wine_get_version)(void);
+    const char * (CDECL *wien_get_version)(void);
 
     InitCommonControls();
 
-    wine_get_version = (void *)GetProcAddress( GetModuleHandleA("ntdll.dll"), "wine_get_version" );
-    if (wine_get_version) strcat( name, wine_get_version() );
+    wien_get_version = (void *)GetProcAddress( GetModuleHandleA("ntdll.dll"), "wien_get_version" );
+    if (wien_get_version) strcat( name, wien_get_version() );
     return !ShellAboutA( NULL, name, NULL, 0 );
 }
